dstore.go

```diff
@@ -1,39 +1,142 @@
 package sqlds
 
 import (
-	"database/sql"
+	"errors"
 	"fmt"
+	"log"
+
+	"database/sql"
 
-	ds "github.com/jbenet/go-datastore"
-	dsq "github.com/jbenet/go-datastore/query"
+	ds "gx/ipfs/QmVG5gxteQNEMhrS8prJSmU2C9rebtFuTd3SYZ5kE3YZ5k/go-datastore"
+	dsq "gx/ipfs/QmVG5gxteQNEMhrS8prJSmU2C9rebtFuTd3SYZ5kE3YZ5k/go-datastore/query"
```

Comment:
We usually use non-rewritten paths (`go-go uw`)

Reply:
Whoops, fixing